cli/state.go

```diff
@@ -1638,6 +1639,34 @@ var stateCircSupplyCmd = &cli.Command{
 	},
 }
 
+// TODO: Remove stateCircSupplyCmd
```

Comment:
I'd
* Make this the default output of `circulating-supply`
* Add a `--vm-supply` flag to get the `api.StateCirculatingSupply` one

Reply:
Yeah, sounds like the approx CircSupply isn't going anywhere, so I will do this